src/pal/src/include/pal/thread.hpp

```diff
@@ -413,6 +412,7 @@ namespace CorUnix
           , m_fInPal(TRUE)
 #endif // FEATURE_PAL_SXS
         {
+            SetLastError(0);
```

Comment:
Thanks.  Unnecessary, and also the source of the PAL test failure, as the test wasn't expected errno to be reset to 0 here.